elide-async/src/test/java/com/yahoo/elide/async/service/AsyncQueryThreadTest.java

```diff
@@ -54,66 +57,40 @@ public void testAsyncQueryCleanerThreadSet() {
     }
 
     @Test
-    public void testProcessQueryJsonApi() {
+    public void testProcessQueryJsonApi() throws NoHttpResponseException, URISyntaxException {
         String query = "/group?sort=commonName&fields%5Bgroup%5D=commonName,description";
+        String id = "edc4a871-dff2-4054-804e-d80075cf827d";
         ElideResponse response = mock(ElideResponse.class);
 
         when(queryObj.getQuery()).thenReturn(query);
+        when(queryObj.getId()).thenReturn(id);
```

Comment:
Don't mock queryObj

Reply:
I had to mock the queryObj for doing Mockito verify calls.